tests/datetime-spec.js

```diff
@@ -730,4 +730,19 @@ describe( 'Datetime', function(){
         assert.equal( thirdMonth, 'Mrt' );
         assert.equal( fifthMonth, 'Mei' );
     });
+
+    it( 'updates display format when dateFormat prop changes', function() {
+		createDatetime({ dateFormat: 'YYYY-MM-DD', timeFormat: false, defaultValue: moment() });
```

Comment:
This indentation is way off, it should match the indentation used in the rest of the file. I usually have this problem too, something is weird.

I usually copy the space used as indentation for a line with correct indentation.